x/emissions/abci.go

```diff
@@ -75,16 +71,24 @@ func DistributeObserverRewards(
 	ctx sdk.Context,
 	amount sdkmath.Int,
 	keeper keeper.Keeper,
-	slashAmount sdkmath.Int,
 ) error {
-
+	params, found := keeper.GetParams(ctx)
+	if !found {
+		return errors.New("params not found")
```

Comment:
Maybe we can provide the param object in argument since it is used in the calling function

Also we should add test case here if we keep it this way